### PR TITLE
issue #33452 fix. Save buttons in admin are too close together on medium screen sizes

### PR DESCRIPTION
**Pull request by AtmegaBuzz**
before: 
image

after:
image

since this is my first contribution to django, if there is something wrong or there is better way to write this code please tell me. i want to learn and contribute more here.

### CODE REVIEW

File: django/contrib/admin/static/admin/css/base.css [hunk 1]
```diff
@@ -502,6 +503,7 @@ a.button {
     background: var(--button-hover-bg);
 }
 
+
```
Comment (felixxm) [inline django/contrib/admin/static/admin/css/base.css:506]: Please revert unrelated changes.

File: django/contrib/admin/static/admin/css/responsive.css [hunk 2]
```diff
@@ -456,6 +456,13 @@ input[type="submit"], button {
     pre.literal-block {
         overflow: auto;
     }
+
+        
+    .submit-row input{
+        margin-top: 5px !important;
+        margin-bottom: 5px !important;
+    }
+
```
Comment (felixxm) [inline django/contrib/admin/static/admin/css/responsive.css:465]: IMO, `!important` is not necessary.

What do you think about decreasing `padding` instead? For example:
```css
.submit-row input {
  padding: 5px 10px;
}
```

File: django/contrib/admin/static/admin/css/responsive.css [hunk 3]
```diff
@@ -888,6 +895,7 @@ input[type="submit"], button {
     }
 
     .login .submit-row {
+        margin: 0;
```
Comment (felixxm) [inline django/contrib/admin/static/admin/css/responsive.css:898]: This change is unrelated, please revert.

**Review by felixxm — COMMENTED** [review]
@AtmegaBuzz Thanks for this patch :+1: I left comments.

File: django/contrib/admin/static/admin/css/responsive.css (hunk 3 above)
Comment (AtmegaBuzz) [reply django/contrib/admin/static/admin/css/responsive.css:898]: sure

File: django/contrib/admin/static/admin/css/responsive.css (hunk 2 above)
Comment (AtmegaBuzz) [reply django/contrib/admin/static/admin/css/responsive.css:465]: thnkyou sir for review, reduced padding is also not working. and idk why the by removing !important the changes is not reflecting.

Comment (felixxm) [reply django/contrib/admin/static/admin/css/responsive.css:465]: It works for me :thinking: 
Before:
image

After:
image

Comment (AtmegaBuzz) [reply django/contrib/admin/static/admin/css/responsive.css:465]: but sir its not working if we have a delete btn 
image

Comment (felixxm) [reply django/contrib/admin/static/admin/css/responsive.css:465]: True, I'd play with `submit-row` and `deletelink` `margin` and `padding` attributes to keep them in a single line.

Comment (AtmegaBuzz) [reply django/contrib/admin/static/admin/css/responsive.css:465]: ok understood , it should be in single line. i revert all changes and again resolve it as you said
